File: PreviousCode/Python/Team.py
class Team:
    def __init__(self, newRank, newName, newVenue, newCity):
        self.rank = newRank
        self.name = newName
        self.venue = newVenue
        self.city = newCity

    def __str__(self):
        result = f"{self.rank} {self.name}"
        return result

File: NewCode/JavaScript/src/Team.js
class Team { // eslint-disable-line no-unused-vars
  constructor (newRank = 0, newName = 'unnamed', newVenue = 'place', newCity = 'city') {
    this.rank = newRank
    this.name = newName
    this.venue = newVenue
    this.city = newCity
  }

  toString () {
    let result = `${this.rank} ${this.name}`
    return result
  }
}
